### DISCUSSION TITLE
Backup failed because The dump process failed with exitcode 2 : Misuse of shell builtins : mysqldump: Got error: 2004: "Can't create TCP/IP socket (10106 "Unknown error")" when trying to connect

### DISCUSSION THREAD

**Discussion by joseAlex27** [spatie/laravel-backup#1097] [Q&A]
Backup failed because The dump process failed with exitcode 2 : Misuse of shell builtins : mysqldump: Got error: 2004: "Can't create TCP/IP socket (10106 "Unknown error")" when trying to connect

pls help with this issue

**Comment by rhalp10** [comment]
> Backup failed because The dump process failed with exitcode 2 : Misuse of shell builtins : mysqldump: Got error: 2004: "Can't create TCP/IP socket (10106 "Unknown error")" when trying to connect
> 
> pls help with this issue

try this if this work for you:
**config/database.php**
```
'mysql' => [
	'dump' => [
               'dump_binary_path' => 'C:\\wamp64\\bin\\mysql\\mysql8.0.18\\bin', // only the path, so without `mysqldump` or `pg_dump`
               'use_single_transaction',
               'timeout' => 60 * 5, // 5 minute timeout
               // 'exclude_tables' => ['table1', 'table2'],
               'add_extra_option' => ' --column-statistics=0', 
            ] , 
]

```
[Backup failed because The dump process failed with exitcode 2 : Misuse of shell builtins : mysqldump: Got error: 2004: "Can't create TCP/IP socket (10106 "Unknown error")" when trying to connect](https://github.com/Laravel-Backpack/BackupManager/issues/55#issuecomment-465981423)

**Reply by khalidJA** [reply]
@rhalp10  not worked for me > same error

**Reply by PhHitachi** [reply]
any solution here?

**Reply by alcarazolabs** [reply]
same problem, laravel 8. I have mysql bin variable in my system path also,

**Reply by AmrSubZero** [reply]
Same `Laravel 8.60.0` at `9/9/2021`

**Reply by alcarazolabs** [reply]
This works only in linux servers. I can confirm it.. I used ubuntu 20.04 and worked not mods needed
